router/router.go

```diff
@@ -195,7 +195,7 @@ func New(cfg *config.Configuration, rateConvertor *currency.RateConverter) (r *R
 	r.MetricsEngine = metricsConf.NewMetricsEngine(cfg, openrtb_ext.CoreBidderNames(), syncerKeys, moduleStageNames)
 	shutdown, fetcher, ampFetcher, accounts, categoriesFetcher, videoFetcher, storedRespFetcher := storedRequestsConf.NewStoredRequests(cfg, r.MetricsEngine, generalHttpClient, r.Router)
 
-	analyticsRunner := analyticsBuild.New(&cfg.Analytics)
+	analyticsRunner := analyticsBuild.New(cfg.Analytics)
```

Comment:
Why is this no longer a pointer? `cfg.Analytics` could be nil if there aren't any analytics modules enabled.